tools/cli/go-whisk-cli/commands/util.go

```diff
@@ -285,8 +285,26 @@ func printSummary(collection interface{}) {
     }
 }
 
+func Len(collection interface{}) func() int {
```

Comment:
@dubeejw , is your comments means this kind of function? I thought perhaps the action array should implement the sort interfaces, right?